std_msgs/Header header
GroupState[] groups
int32 total_groups